src/features/duplicates/store.tsx

```diff
@@ -2,51 +2,60 @@ import { ZetkinPerson } from 'utils/types/zetkin';
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { RemoteList, remoteList } from 'utils/storeUtils';
 
-export interface ZetkinDuplicate {
-  duplicatePersons: ZetkinPerson[];
+export interface PotentialDuplicate {
+  duplicates: ZetkinPerson[];
   dismissed: string | null;
   id: number;
   merged: string | null;
   organization: { id: number; title: string };
   status: 'merged' | 'dismissed' | 'pending';
 }
 
-export interface DuplicatesStoreSlice {
-  duplicatesList: RemoteList<ZetkinDuplicate>;
+export interface PotentialDuplicatesStoreSlice {
+  potentialDuplicatesList: RemoteList<PotentialDuplicate>;
 }
 
-const initialState: DuplicatesStoreSlice = {
-  duplicatesList: remoteList(),
+const initialState: PotentialDuplicatesStoreSlice = {
+  potentialDuplicatesList: remoteList(),
 };
 
-const duplicatesSlice = createSlice({
+const potentialDuplicatesSlice = createSlice({
   initialState,
-  name: 'duplicates',
+  name: 'potentialDuplicates',
   reducers: {
-    dismissedDuplicate: (state, action: PayloadAction<ZetkinDuplicate>) => {
-      const duplicate = action.payload;
-      const item = state.duplicatesList.items.find(
-        (item) => item.id === duplicate.id
+    dismissedPotentialDuplicate: (
```

Comment:
I believe this should be something along the lines of `potentialDuplicateDismissed` or even just `duplicateUpdated`.